export/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java

```diff
@@ -90,5 +90,11 @@
   )
   public static boolean verbose = false;
 
+  @Parameter(
+      names = {"--monitoring_project_id"},
+      description = "Stackdriver monitoring project id"
```

Comment:
Can we add "must be the same as the project id in which the adapter is running" to the end of the flag comment for both import/export adapters? Otherwise users might think they can send events to other projects.

Reply:
Ok